js/sales.js

```diff
@@ -0,0 +1,109 @@
+    }
+};
+
+var seattleCenter = {
+    name: 'Seattle Center',
+    minHourlyCustomers: 11,
+    maxHourlyCustomers: 38,
+    averageCookiesPerCustomer: 3.7, 
+    randomCustomersPerHour: function() {
+        return Math.ceil(Math.random() * (this.maxHourlyCustomers - this.minHourlyCustomers)) + this.minHourlyCustomers;
+    }
+};
+
+var capitolHill = {
+    name: 'Capitol Hill',
+    minHourlyCustomers: 20,
+    maxHourlyCustomers: 38,
+    averageCookiesPerCustomer: 2.3, 
+    randomCustomersPerHour: function() {
+        return Math.ceil(Math.random() * (this.maxHourlyCustomers - this.minHourlyCustomers)) + this.minHourlyCustomers;
+    }
+};
+
+var alki = {
+    name: 'Alki',
+    minHourlyCustomers: 2,
+    maxHourlyCustomers: 16,
+    averageCookiesPerCustomer: 4.6, 
+    randomCustomersPerHour: function() {
+        return Math.ceil(Math.random() * (this.maxHourlyCustomers - this.minHourlyCustomers)) + this.minHourlyCustomers;
+    }
+};
+
+function calculateSimulatedCookiesPerHour(shopLocation){
+    //using military time
+    var openHours = 20 - 6;
+    var simulatedCookiesPerHour = [];
+    for(var i = 0; i < openHours; i++){
+        simulatedCookiesPerHour[i] = Math.floor(shopLocation.randomCustomersPerHour() * shopLocation.averageCookiesPerCustomer);
+    }
+    return simulatedCookiesPerHour;
+}
+
+function createStoreLocationHTMLSections(){
+    var cookieLocations = [firstPike, seatacAirport, seattleCenter, capitolHill, alki];
+    for(var i = 0; i < cookieLocations.length; i++){
+        var storeLocation = cookieLocations[i];
+        storeLocation.simulatedCookiesPerHour = calculateSimulatedCookiesPerHour(storeLocation);
+        createH2Element(storeLocation);
+        createULElement(storeLocation);
+        createLIElements(storeLocation);
+    }
+}
+
+function createH2Element(location){
+    var newH2Element = document.createElement('h2');
+    newH2Element.setAttribute('class', location.name.replaceAll(" ", "-"));
+    newH2Element.textContent = location.name;
+    document.querySelector('body').append(newH2Element);
+}
+
+function createULElement(location){
+    var h2Element = document.getElementsByClassName(location.name.replaceAll(" ", "-"))[0];
+    var newULElement = document.createElement('ul');
+    newULElement.setAttribute('class', location.name.replaceAll(" ", "-"));
+    h2Element.insertAdjacentElement('afterend', newULElement);
+}
+
+function createLIElements(location){
+    var h2Element = document.getElementsByClassName(location.name.replaceAll(" ", "-"))[1];
+    var openingHour = 6;
+    for(var i = 0; i < location.simulatedCookiesPerHour.length; i++){
+        var newLIElement = document.createElement('li');
+        var hourSoldCookies = location.simulatedCookiesPerHour[i];
+        if(openingHour + i < 12){
+            //1am-11am
+            newLIElement.textContent = (openingHour + i) + "am: " + hourSoldCookies + " cookies";
```

Comment:
We're being a bit repetitive with how we're assigning the `textContent` property. We could make this code more DRY by perhaps doing something like:
![Capture](https://user-images.githubusercontent.com/20998142/84962973-ea54fd00-b0bc-11ea-8445-14e95c5deef6.PNG)